src/features/account/repositories/account-repository/account-repository.ts

```diff
@@ -0,0 +1,13 @@
+import { database } from '@/shared/infra/database/database.js';
+
+export class AccountRepository {
+  async getAccounts(userId: string) {
+    const userAccounts = await database.account.findMany({
+      where: {
+        userId: userId,
+      },
+    });
+
+    return userAccounts || [];
```

Comment:
Quando o find não encontra registros ele já retorna um array vazio, acho que ai já pode retornar direto o `userAccounts`
```suggestion
    return userAccounts;
```

Acho que ajustando isso, deve resolver o alerta do coverage que gerou no build, que não cobriu 100% do código.